llvm/test/tools/llvm-objcopy/ELF/change-section-lma.test

```diff
@@ -1,32 +1,45 @@
 # RUN: yaml2obj %s -o %t
 # RUN: llvm-objcopy --change-section-lma *+0x20 %t %t2
-# RUN: llvm-readelf -l %t2 | FileCheck %s --check-prefix=CHECK-PLUS
+# RUN: llvm-readelf --program-headers %t2 | FileCheck %s --check-prefix=CHECK-PLUS-PROGRAMS
+# RUN: llvm-readelf --section-headers %t2 | FileCheck %s --check-prefix=CHECK-PLUS-SECTIONS
```

Comment:
You could simplify this and the similar below down to a single call with both options specified.

Reply:
Yes the original version had them in one command but I "simplified" it by having two separate commands. I think it makes it clearly to see what is being checked.